### PR TITLE
Implemented tests

### PR DESCRIPTION
**Pull request by Ilithor**

### CODE REVIEW

File: src/components/login/LoginForm.js [hunk 1]
```diff
@@ -80,3 +80,15 @@ const LoginForm = ({
 );
 
 export default withStyles(style)(LoginForm);
+
+/** Various props that control how the user interacts with
+ *  the login form
+ * @typedef ILoginFormComponentProps
```
Comment (jeffersoneagley) [inline src/components/login/LoginForm.js]: You should also change the above type notation for LoginForm to 
```
* @param {ILoginFormComponentProps} props
```

Comment (Ilithor) [reply src/components/login/LoginForm.js]: Fixed

Comment (jeffersoneagley) [reply src/components/login/LoginForm.js]: I think you forgot to push

File: src/components/login/LoginForm.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ import style from '../../style/style';
 /** Various props that control how the user interacts with
  *  the login form
  * @type {React.FunctionComponent}
- * @param {object} props
+ * @param {ILoginFormComponentProps} props
  * @param {any} props.error
  * @param {object} props.classes
```
Comment (jeffersoneagley) [inline src/components/login/LoginForm.js]: You don't need these items now that ILoginFormComponentProps includes them

Comment (Ilithor) [reply src/components/login/LoginForm.js]: Fixed

File: src/components/profile/user/UserProfileDisplay.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ import {
 
 /** View component for displaying the user's profile
  * @type {React.FunctionComponent}
- * @param {object} props
+ * @param {IUserProfileDisplayComponentProps} props
  * @param {object} props.classes
  * @param {string} props.handle
  * @param {object} props.bio
```
Comment (jeffersoneagley) [inline src/components/profile/user/UserProfileDisplay.js]: 27 through 29 are implied as part of IUserPRofileDisplayComponentProps

File: src/components/login/LoginForm.js [hunk 4]
```diff
@@ -9,19 +9,7 @@ import CircularProgress from '@material-ui/core/CircularProgress';
...
- * @param {any} props.error
- * @param {object} props.classes
- * @param {string} props.email
- * @param {string} props.password
- * @param {React.ChangeEventHandler} props.handleSubmit
- * @param {React.ChangeEventHandler} props.handleChange
- * @param {boolean} props.isLoading
- */
-const LoginForm = ({
+export const LoginForm = ({
```
Comment (jeffersoneagley) [inline src/components/login/LoginForm.js]: `@type {ILoginFormComponentProps} props`?

Comment (Ilithor) [reply src/components/login/LoginForm.js]: Fixed

File: src/components/profile/user/UserProfileDisplay.js (hunk 3 above)
Comment (Ilithor) [reply src/components/profile/user/UserProfileDisplay.js]: Fixed

**Pull request merged**